infer/src/base/Config.ml

```diff
@@ -497,6 +498,14 @@ and bootclasspath =
     ~exes:CLOpt.[Toplevel; Java]
     "Specify the Java bootclasspath"
 
+and bo_debug =
+  CLOpt.mk_int ~default:0 ~long:"bo-debug"
+    ~exes:CLOpt.[Toplevel] "Debug mode for buffer-overrun checker"
+
+and bo_filtering =
```

Comment:
Infer has a builtin-in [mechanism](https://github.com/facebook/infer/blob/master/infer/src/IR/Localise.ml#L152) for filtering/prioritizing warnings. It would probably be best to re-use that mechanism rather than creating a new one just for buffer overruns.